infrastructure/ansible/site.yml

```diff
@@ -31,8 +31,8 @@
   strategy: free
   roles:
     - java
+    - http_server
     - nginx
-    - dropwizard
```

Comment:
I'm not sure if this is "re-ordering"

Reply:
:grin:  The perils of moving "independent" updates. Here though the commit commentary was incomplete. This is both a fix and a re-order. Running this playbook yields an error right now. The 'dropwizard' role DNE. The role name comes from the sub-folder name in the 'roles' folder: https://github.com/triplea-game/triplea/tree/master/infrastructure/ansible/roles, so this must be 'http_server'

I've updated the commit comment to reflect the correction.

Reply:
I figured something like that, thanks for clarifying ^^

Reply:
:+1: